lib/errorHandler.ts

```diff
@@ -0,0 +1,26 @@
+import { NextApiResponse } from "next";
+
+export function errorHanlder(err: unknown, res: NextApiResponse) {
```

Comment:
small typo. need to change wherever "`errorHanlder`" was used

```suggestion
export function errorHandler(err: unknown, res: NextApiResponse) {
```